Gopkg.toml

```diff
@@ -12,33 +12,37 @@ required = [
 
 [[constraint]]
   name = "k8s.io/client-go"
-  branch = "release-10.0"
+  branch = "release-11.0"
 
 [[constraint]]
   name = "k8s.io/api"
-  branch = "release-1.13"
+  branch = "release-1.14"
 
 [[constraint]]
   name = "k8s.io/kubernetes"
-  branch = "release-1.13"
+  branch = "release-1.14"
```

Comment:
we could use the kubernetes-1.14.0 tags instead

Reply:
done